### PR TITLE
Add required with multiple present

### PR DESCRIPTION
**Pull request by MaximeRDY**
Hello, 

I've added `required: true` for multiple present. 

Have a good day

### CODE REVIEW

**Comment by MaximeRDY** [comment]
@LeFnord Rubocop 1.7.0 is bugged so I've fixed to 1.6.1 (last version without bug)

**Comment by LeFnord** [comment]
thanks @MaximeRDY  … somehow I can't figure it out, what require means in the context of a response … 🤔

File: Gemfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ group :development, :test do
   gem 'rake'
   gem 'rdoc'
   gem 'rspec', '~> 3.9'
-  gem 'rubocop', '~> 1.0', require: false
+  gem 'rubocop', '1.6.1', require: false
```
Comment (LeFnord) [inline Gemfile:30]: what is the bug?

Comment (MaximeRDY) [reply Gemfile]: They fix in Rubocop 1.8.0. So never mind :-)

**Comment by MaximeRDY** [comment]
> thanks @MaximeRDY … somehow I can't figure it out, what require means in the context of a response … 🤔

In multiple present response, the response look like this: 
```
"responses": {
    "200": {
      "description": "Multiple response",
      "schema":{
        "type":"object",
        "properties":{
          "gender":{
            "$ref":"#/definitions/EnumValues"
          },
          "somethings":{
            "$ref":"#/definitions/Something"
          }
        }
      }
    }
  }
  ```
  
  But here the property `gender` and `somethings` is by definition optional (= can respond `null`) 
  So if you want to specify the property `somethings` cannot respond `null` we need to have a response like this : 
  ```
  "responses": {
    "200": {
      "description": "Multiple response",
      "schema":{
        "type":"object",
        "properties":{
          "gender":{
            "$ref":"#/definitions/EnumValues"
          },
          "somethings":{
            "$ref":"#/definitions/Something"
          }
        },
       "required": ["somethings"]
      }
    }
  }
  ```
  
  So it's why I've added the possibility to have something like :
  ```
  { model: Entities::Something, as: :somethings, is_array: true, required: true }
  ```

**Comment by LeFnord** [comment]
ah ok @MaximeRDY … thanks for explanation … please add also a README entry and I'm happy to mörge it

**Comment by LeFnord** [comment]
sorry, overseen it

**Pull request merged**